README.md

```diff
@@ -56,13 +58,14 @@ When an OdeMapJson message is processed through the jpo-geojsonconverter, a Proc
 [ProcessedMap schema can be found here.](<jpo-geojsonconverter/src/main/resources/schemas/processed-map.schema.json>)
 
 ### ProcessedSpat
+
 When an OdeSpatJson message is processed through the jpo-geojsonconverter, a ProcessedSpat message is created. This message is a single JSON object that contains all of the important information within a SPaT message for matching it to a corresponding ProcessedMap message and for identifying its state. There is no geoJSON component to a ProcessedMap message on its own.

```

Comment:
Should say "There is no geoJSON component to a ProcessedSpat" message on its own

Reply:
addressed